src/main/java/com/cascadebot/cascadebot/commandmeta/CommandContext.java

```diff
@@ -437,7 +437,7 @@ public void sendPagedMessage(List<Page> pages) {
             pageGroup.getPage(pageGroup.getPages()).pageShow(message, pageGroup.getPages(), pageGroup.getPages());
             pageGroup.setCurrentPage(pageGroup.getPages());
         }));
-        channel.sendMessage("\u00A0").queue(sentMessage -> {
+        channel.sendMessage("Paged message loading...").queue(sentMessage -> {
```

Comment:
Why do we need a message here. it doesn't take like any time at all the update the message with pages?

Reply:
Well we need something, this is just the most sensical thing I could think of